### ISSUE TITLE
Invalid Argo CLI download link in Web UI

### ISSUE THREAD

**Issue by yoichiwo7** [argoproj/argo#3762]
## Summary 

The Argo CLI download link in Web UI is invalid which leads to 404 error.

## Diagnostics

What version of Argo Workflows are you running?
2.9.5

The download links of all platforms are invalid. 
```
https://github.com/argoproj/argo/releases/download/latest/argo-linux-amd64
https://github.com/argoproj/argo/releases/download/latest/argo-darwin-amd64
https://github.com/argoproj/argo/releases/download/latest/argo-windows-amd64
```
It looks like `latest` tag in the URL doesn't exist.

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by alexec** [comment]
Good spot. We should remove these out dated links.

**Comment by alexec** [comment]
Available for testing in [v2.10.0-rc1](https://github.com/argoproj/argo/releases/tag/v2.11.0-rc1).
